src/views/v2/events-bar/filters.php

```diff
@@ -13,21 +13,3 @@
  *
  */
 ?>
-<div class="tribe-events-c-events-bar__filters">
```

Comment:
should we remove this file completely? or are you keeping this to replace with filter bar when activated?

Reply:
Was moved to filter-bar.